app/root.tsx

```diff
@@ -1,13 +1,19 @@
 import type { LinksFunction, LoaderFunctionArgs } from '@remix-run/node';
 import { json } from '@remix-run/node';
 import stylesheet from './styles/tailwind.css';
+import faviconAssetUrl from './imgs/favicon.svg';
 import { TopNav } from '~/components/layout/top-nav';
 import { Links, LiveReload, Meta, Outlet, Scripts, ScrollRestoration } from '@remix-run/react';
 import Footer from './components/layout/footer';
 import { getCurrentUser } from '~/modules/session/session.server';
 import { db } from './modules/database/db.server';
 
-export const links: LinksFunction = () => [{ rel: 'stylesheet', href: stylesheet }];
+export const links: LinksFunction = () => {
+  return [
+    { rel: 'icon', type: 'image/svg-xml', href: faviconAssetUrl },
```

Comment:
Curious, what is `icon` used for since we also have a `favicon` already in the public folder?

Reply:
My understanding is the favicon in the public folder has a cach maxAge of 1hr, so if the SVG file in the public folder is changed it may take awhile to update. Importing the favicon from the app folder may immediately provide the latest version of the SVG file for our browser.